### PR TITLE
Demisto word in the repo README

### PR DESCRIPTION
**Pull request by AradCarmi**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/42164)

## Description
Fixes README validation- ignoring the case that the README is in the main repo.

## Must have
- [x] Tests
- [x] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: demisto_sdk/commands/common/tests/readme_test.py [hunk 1]
```diff
@@ -347,6 +347,36 @@ def test_demisto_in_readme(repo):
...
+
+        When
+            - Running verify_demisto_in_readme_content.
+
+        Then
+            - Ensure that the validation not fails.
+    """
+    from pathlib import Path
+
+    readme_path = Path(repo.path) / '*README.md'
```
Comment (dantavori) [inline demisto_sdk/commands/common/tests/readme_test.py:368]:
```suggestion
    readme_path = Path(repo.path) / 'README.md'
```
why with `*`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 * Fixed an issue in **update-release-notes** where suggestion fix failed in validation.
 * Fixed a bug where `.env` files didn't load. @nicolas-rdgs
 * Fixed a bug where **validate** command failed when the *categories* field in the pack metadata was empty for non-integration packs.
+* Fixed a bug where **validate** command failed when the word demisto is in the repo README file.
```
Comment (dantavori) [inline CHANGELOG.md:6]: you will need to handle conflicts in this file, FYI

**Review by dantavori — APPROVED** [review]
:very-nice:

**Comment by AradCarmi** [comment]
> we can also ignore when we use "demisto lock" integration

For packs we are using the .pack-ignore, no need to specify this in the sdk valdition.

**Pull request merged**